test/unit/org/apache/cassandra/repair/FuzzTestBase.java

```diff
@@ -654,6 +667,7 @@ static class Cluster
 
             // We run tests in an isolated JVM per class, so not cleaing up is safe... but if that assumption ever changes, will need to cleanup
             Stage.ANTI_ENTROPY.unsafeSetExecutor(orderedExecutor);
+            Stage.MISC.unsafeSetExecutor(orderedExecutor);
```

Comment:
one line of code did this rather than `Stage.ANTI_ENTROPY`... not sure why we do MISC rather than ANTI_ENTROPY given this is repair... but figured best not to change it...  For this test they are the same executor so doesn't matter.